services/scabbard/libscabbard/src/store/scabbard_store/diesel/operations/add_consensus_action.rs

```diff
@@ -25,10 +25,11 @@ use splinter::service::FullyQualifiedServiceId;
 
 use crate::store::scabbard_store::diesel::{
     models::{
-        CoordinatorContextModel, CoordinatorNotificationModel, CoordinatorSendMessageActionModel,
-        InsertableConsensusActionModel, ParticipantContextModel, ParticipantNotificationModel,
-        ParticipantSendMessageActionModel, UpdateCoordinatorContextActionModel,
-        UpdateCoordinatorContextActionParticipantList, UpdateParticipantContextActionModel,
+        InsertableTwoPcConsensusActionModel, TwoPcCoordinatorContextModel,
+        TwoPcCoordinatorNotificationModel, TwoPcCoordinatorSendMessageActionModel,
```

Comment:
I'm not sure of a good alternative prefix for these, but `TwoPc` seems a little off to me. Since we can't start with a number, perhaps we could simply have these structs live in a `models::two_phase_commit` module, and retain their non-prefixed names.

Thoughs? (@vaporos, @agunde406)